dev-tools/generate_notice.py

```diff
@@ -82,16 +91,16 @@ def create_notice(filename, beat, copyright, licenses):
 
     cwd = os.getcwd()
     notice = os.path.join(cwd, "NOTICE")
-    licenses = []
+    license_globs = []
 
     for root, dirs, files in os.walk(args.vendor):
```

Comment:
Can you filter out the `.git` dir? I see it searching inside there when I run it and it really shouldn't be looking in there.